src/main/resources/import.sql

```diff
@@ -9,6 +9,9 @@ INSERT INTO user_details (name,password,user_role) VALUES ('Kazek', 'sadasda','C
 INSERT INTO user_details (name,password,user_role) VALUES ('John', 'admin123','Client');
 INSERT INTO user_details (name,password,user_role) VALUES ('Dlugos','password666','Client');
 
+INSERT INTO address (street,number,postalCode,city,country) VALUES ('Borownice', '12','33-350','Piwniczna','Poland');
```

Comment:
What's the plan? Do we want to make some multi insert here or add some data via java code?